masonry/src/widget/portal.rs

```diff
@@ -245,11 +245,8 @@ impl<W: Widget> Widget for Portal<W> {
 
         match event {
             PointerEvent::MouseWheel(delta, _) => {
-                self.set_viewport_pos_raw(
-                    portal_size,
-                    content_size,
-                    self.viewport_pos + Vec2::new(delta.x, delta.y),
-                );
+                let delta = Vec2::new(delta.x * -10., delta.y * -10.);
```

Comment:
What are these magic numbers?

Reply:
Plugging the default values as deltas makes scrolling way too slow.

You're right that using magic numbers is a poor solution, though. I'll use a constant for now.